ci/integration.yml

```diff
@@ -0,0 +1,16 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: 1.8-alpine
```

Comment:
Could we not use a smaller container (eg vanilla alpine) as this task is currently a no-op?

Reply:
Good idea, updated to alpine:3.4 (we already use this tag elsewhere)